cronos-play/getting-started_unreal/quick-start/erc1155.md

```diff
@@ -0,0 +1,6 @@
+# ERC1155
+
+We support the following ERC1155 contract functions:
```

Comment:
do those functions have good description on functionality / args inside the Unreal editor?

Reply:
yes, they have... ok to add them back.

Reply:
Done